pom.xml

```diff
@@ -60,18 +59,6 @@
             <artifactId>core-modules</artifactId>
             <version>${mule.service.weave.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-model</artifactId>
-            <version>${mavenVersion}</version>
-        </dependency>
-
-        <!--TODO: DELETE ME FER-->
-        <dependency>
-            <groupId>com.googlecode.slf4j-maven-plugin-log</groupId>
-            <artifactId>slf4j-maven-plugin-log</artifactId>
-            <version>1.0.0</version>
-        </dependency>
```

Comment:
@fernandothea could you please confirm if it is safe to delete this ?

Reply:
👀